app/scraper/sources/face_book.py

```diff
@@ -162,7 +170,11 @@ def scrape(self, current_people):
                     person['birth_day'] = int(birth_day)
                 person['major'] = trivia.pop() if trivia[-1] in MAJORS else None
                 if person['major'] and person['major'] in MAJOR_FULL_NAMES:
-                    person['major'] = MAJOR_FULL_NAMES[person['major']]
+                    if person['major'] != VISITING_INTERNATIONAL:
```

Comment:
To decrease the branch complexity of this statement, I think it would be better to assign `person['major']` first, and then add a subsequent check for whether it's `Visiting International Program` and override appropriately.